pkg/tests/end_to_end_tests/functions_test.go

```diff
@@ -21,6 +21,33 @@ import (
 	"github.com/timescale/promscale/pkg/prompb"
 )
 
+func TestMetricView(t *testing.T) {
```

Comment:
nit: I would add `func` as a suffix in the test func name to indicate it's a test for the SQL func.